README.md

```diff
@@ -23,28 +23,28 @@ update_julia()
 julia> update_julia()
 installing the latest version of julia: 1.7.0
 ...
-Success! `julia-1.7.0` & `julia-1.7` & `julia` now to point to 1.7.0
+Success! `julia-1.7.0` & `julia-1.7` & `julia` now to point to 1.7.0.
```

Comment:
These `.`s are intentionally omitted because I value the clarity that the version string does not end with a `.` more than printing complete, well-punctuated sentences. (The readme is also copied from a REPL, so to change this output would require changing the code that generated it.)